### PR TITLE
[luci] Tidy RemoveRedundantTransposePass comments

### PR DESCRIPTION
**Pull request by seanshpark**
This will tidy RemoveRedundantTransposePass diagram comments.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

File: compiler/luci/pass/src/RemoveRedundantTransposePass.cpp [hunk 1]
```diff
@@ -90,30 +90,26 @@ bool remove_consecutive_transpose_function(luci::CircleTranspose *target_node)
...
  *  BEFORE
  *         |
  *   [CircleNode]     [CircleConst]
- *    (main_node)      (main_perm)
- *         \               /
+ *         |           (pred_perm)
+ *          \              /
  *         [CircleTranspose]  [CircleConst]
- *            (pred_node)      (pred_perm)
+ *            (pred_node)     (target_perm)
```
Comment (struss) [inline compiler/luci/pass/src/RemoveRedundantTransposePass.cpp:101]: https://github.com/Samsung/ONE/blob/e2014ebf11b6e43c049ad7c46485a1734513e88e/compiler/luci/pass/src/RemoveRedundantTransposePass.cpp#L44-L46
If we change this comment from `perd_perm` to `target_perm`, then this should be changed to matching with comment.

File: compiler/luci/pass/src/RemoveRedundantTransposePass.cpp [hunk 2]
```diff
@@ -90,30 +90,26 @@ bool remove_consecutive_transpose_function(luci::CircleTranspose *target_node)
...
 namespace luci
 {
+
 /**
  *  BEFORE
  *         |
  *   [CircleNode]     [CircleConst]
- *    (main_node)      (main_perm)
- *         \               /
+ *         |           (pred_perm)
```
Comment (struss) [inline compiler/luci/pass/src/RemoveRedundantTransposePass.cpp:98]: https://github.com/Samsung/ONE/blob/e2014ebf11b6e43c049ad7c46485a1734513e88e/compiler/luci/pass/src/RemoveRedundantTransposePass.cpp#L48-L50
If we change this comment from `main_perm`to `pred_perm`, then this should be changed to matching with comment.

File: compiler/luci/pass/src/RemoveRedundantTransposePass.cpp (hunk 1 above)
Comment (seanshpark) [reply compiler/luci/pass/src/RemoveRedundantTransposePass.cpp:101]: I'll post another PR as thic PR is to fix comments.

Comment (seanshpark) [reply compiler/luci/pass/src/RemoveRedundantTransposePass.cpp:101]: #6153 posted

**Review by struss — APPROVED** [review]
As Commented contents will fixed on #6153 , LGTM!

**Pull request merged**